app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -122,6 +128,17 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener {
         mIsInCountdownMode = false
         mCountdownTime = 0
 
+        mBurstHandler = Handler()
+        mBurstRunnable = object : Runnable {
+            override fun run() {
+                mBurstEnabled = true
+                shutter.beGone()
```

Comment:
It would be nice if all these .beGone() and .beVisible() etc were put inside a function like toggleBurstModeButton() or something, makes the code a bit cleaner to have a function for that; we're already doing those toggle functions for all the buttons so we might as well continue